PWGCF/Flow/Core/FlowJHistManager.cxx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// Header files.
+
+// O2 headers.
+
+// O2 Physics headers.
+#include "PWGCF/Flow/Core/FlowJHistManager.h"
+
+// Namespaces.
+using namespace o2;
+using namespace o2::framework;
+
+namespace o2::analysis::PWGCF
+{
+/// \brief Create the histograms in the QA registry.
+void FlowJHistManager::CreateHistQA()
```

Comment:
From the Jyväskylä group you already have a `PWGCF/JCorran/Core/JHistManager.cxx`
Can that be expanded to incorporate this functionality?

Reply:
JHistManager from JCorran is based on the AliPhysics format. This class has O2 histogram implementations and hence I think we need to add functions from JCorran file to this one. I'll discuss this with Jasper.

Reply:
The JHistManager will most likely go given that it's not very compatible with the O2 output objects, and that the TNamed functionality that enables it to be written has never been properly implemented.

Also, @MaximVirta we should probably get rid of the `GetCentBin()` entirely because it's not very flexible to have the binning hardcoded. The cent index should be supplied by the top level task, like I did in my previous commits. This is obtained from a configurable axis.